### PR TITLE
PLATUI-848: Remove Play 2.5 support

### PR DESCRIPTION
**Pull request by matthewmascord**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (JoPintoPaul) [inline LICENSE:1]: Out of interest, why is this now required?

File: README.md [hunk 2]
```diff
@@ -17,6 +17,5 @@ PLAY_VERSION=2.X sbt "project itServer" run
 ## Installation
 
 ```sbt
-// replace suffix play-26 with play-25 if using with Play 2.5
 libraryDependencies += "uk.gov.hmrc" %% "play-nunjucks-viewmodel" % "x.y.z-play-26"
```
Comment (JoPintoPaul) [inline README.md:20]: To confirm, we're not supporting Play 2.7 at the moment? I'm fine with that, just wanting to confirm.

File: project/project/PlayCrossCompilation.scala [hunk 3]
```diff
@@ -1,24 +1,22 @@
...
 import Keys._
 import uk.gov.hmrc.playcrosscompilation.AbstractPlayCrossCompilation
-import uk.gov.hmrc.playcrosscompilation.PlayVersion.{Play25, Play26}
+import uk.gov.hmrc.playcrosscompilation.PlayVersion.Play26
 
-object PlayCrossCompilation extends AbstractPlayCrossCompilation(defaultPlayVersion = Play25) {
+object PlayCrossCompilation extends AbstractPlayCrossCompilation(defaultPlayVersion = Play26) {
 
   lazy val version: String = playVersion match {
-    case Play25 => "2.5.19"
```
Comment (JoPintoPaul) [inline project/project/PlayCrossCompilation.scala:9]: Did v2.0.0 of the cross-compile plugin kill 2.5?

Comment (matthewmascord) [reply project/project/PlayCrossCompilation.scala:9]: Yep

File: README.md (hunk 2 above)
Comment (matthewmascord) [reply README.md:20]: Correct, that's a bigger piece.

File: LICENSE (hunk 1 above)
Comment (matthewmascord) [reply LICENSE:1]: No idea but required by sbt-auto-build.

**Review by JoPintoPaul — APPROVED** [review]

**Pull request merged**